src/deployment/bicep-templates/feature-flags.bicep

```diff
@@ -0,0 +1,20 @@
+param location string
+
+var suffix = uniqueString(resourceGroup().id)
+var appConfigName = 'app-config-${suffix}'
+
+resource featureFlags 'Microsoft.AppConfiguration/configurationStores@2022-05-01' = {
+  name: appConfigName
+  location: location
+  sku:{
```

Comment:
Managed identity is better than connection string:

  identity: {
    type: 'SystemAssigned'
  }